NameLayerMain/src/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -109,15 +123,19 @@ public boolean deleteGroup(String groupName){
 		
 		group.setDisciplined(true);
 		group.setValid(false);
-		
-		if (NameLayerPlugin.isMercuryEnabled()){
+		if (savetodb){
+			groupManagerDao.deleteGroup(groupName);
 			String message = "delete " + groupName;
-			Mercury.invalidateGroup(message);
+			Mercury.message(message);
```

Comment:
Checking if mercury is enabled before triggering it is critical for servers that don't run Mercury. Just FYI. Please add that check back in :).